### PR TITLE
Bump submodule, add WHOX, reorganize groups in the spec list

### PR DESCRIPTION
**Pull request by progval**

### CODE REVIEW

**Comment by emersion** [comment]
soju can be added as both a client and server supporting WHOX. Other than that, LGTM!

**Comment by progval** [comment]
Ah fair, I didn't check bouncers.

Though I don't see anything about soju explicitly WHOX as a client, it's just forwarding requests. Does this warrant a green tick here?

**Comment by emersion** [comment]
Bouncers are just proxies, so not sure how we could better support the extension? e.g. how can a bouncer support MONITOR, away-notify, account-tag, other than properly forwarding the data?

**Comment by progval** [comment]
Added both.

File: _irc/index.md [hunk 1]
```diff
@@ -260,6 +266,31 @@ covers the `label` tag, and how clients should send and will receive
 labels.
 
 
+## Listing Users
+
+IRCv3 specifications extend traditional queries to carry more information about
+other users in a given channel.
+
+The [`multi-prefix` spec]({{site.baseurl}}/specs/extensions/multi-prefix.html)
+details the exact messages these changes apply to and how exactly it's used.
```
Comment (emersion) [inline _irc/index.md:275]: This sentence was originally after the next one (which acted as a kind of introduction). Now this reads a bit weird. Maybe we could rejigger the text a bit?

**Review by emersion — APPROVED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -45,7 +45,8 @@ You should make sure your manually installed local versions match that of [GitHu
 1. Update the `spec` submodule. If the spec's a draft, confirm that it has the `work-in-progress: true` tag in the yaml header of the file.
 2. List the spec on `_irc/index.md`
 3. Add the spec to `_data/irc_versions.yml` - the name here is what's used in the support lists. Look at the `hide-if-no-support` and `hide-on-servers` options, they affect how the support tables are generated. Make sure you put it in the right place sorting-wise.
-4. Add the spec to `_data/specs.yml` - the name here is what's used in the registry. Maybe some other places? I need to merge this and the above file sometime.
+4. Add the spec to `_data/specs.yml` - the name here is what's used in the registry.
+5. Add relevant numerics/capabilities/isupport tokens/... to `_data/registry.yml`
 5. Add the relevant support to the data files!
```
Comment (jwheare) [inline README.md:50]: Misnumbered now.

**Review by jwheare — CHANGES_REQUESTED** [review]
Let's add the 354 numeric to the registry as well

**Comment by jwheare** [comment]
Thank you!

**Pull request merged**